modules/core/src/shaderlib/project/viewport-uniforms.js

```diff
@@ -35,11 +37,38 @@ const IDENTITY_MATRIX = [1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1];
 const DEFAULT_PIXELS_PER_UNIT2 = [0, 0, 0];
 const DEFAULT_COORDINATE_ORIGIN = [0, 0, 0];
 
-// TODO: Find the best value for this to maximize accuracy
-const LNGLAT_EXPERIMENTAL_ZOOM_THRESHOLD = 12;
+// Based on viewport-mercator-project/test/fp32-limits.js
+const LNGLAT_AUTO_OFFSET_ZOOM_THRESHOLD = 12;
```

Comment:
Maybe move this constant to `./constants` next to the projection modes.

Reply:
No